nextmv/nextmv/cloud/manifest.py

```diff
@@ -299,7 +301,7 @@ class ManifestPython(BaseModel):
     'requirements.txt'
     """
 
-    pip_requirements: Optional[str] = Field(
+    pip_requirements: Optional[Union[str, list]] = Field(
```

Comment:
Can you specify list[str] to be more precise?

Reply:
Yeah, I thought about it but for some reason didn't add it. :confused: 
I added it. Thanks for the reminder! :heart: